server/config.ts

```diff
@@ -15,30 +15,36 @@ const nonSensitiveSharedConfig = {
 }
 
 const keysToPullFromEnv = [
-  'SophtronClientId',
-  'SophtronClientSecret',
-
-  'UcpAuthClientId',
-  'UcpAuthClientSecret',
-  'UcpAuthEncryptionKey',
-
-  'CryptoKey',
-  'CryptoIv',
-
-  'Demo',
-  'DefaultProvider',
-  'Port',
-
-  'HostUrl',
-  'WebhookHostUrl',
-  'LogLevel',
-
-  'Env',
-
-  'RedisServer',
-  'RedisCacheTimeSeconds',
-  'ResourcePrefix',
-  'ResourceVersion'
+  "CryptoKey",
+  "CryptoIv",
+
+  "AuthProvider", //supported: ucp, local, sophtron
+  // fill the corresponding credentials below if the auth provider is to be used
+
+  "LocalAuthEncryptionKey", // shared encryption key with caller to encrypt the auth phraze
```

Comment:
It seems like we need something added to the readme describing when you would want to use these and how to set them up